Client/Frontend/Browser/URLBarView.swift

```diff
@@ -299,25 +300,25 @@ class URLBarView: UIView {
     }
 
     func updateTabCount(count: Int) {
-
         // make a 'clone' of the tabs button
         let newTabsButton = InsetButton()
+        newTabsButton.setTranslatesAutoresizingMaskIntoConstraints(false)
```

Comment:
I think this line is redundant since your're using `snp_makeConstraints` on `newTabsButton` below, right? I'm fine with keeping it if you think it helps makes things clearer -- just want to be sure I understand how this all works.